### DISCUSSION TITLE
Why is /docs/ on github.io 301 redirecting a subdomain?

### DISCUSSION THREAD

**Discussion by terrafrost** [community/community#22241] [Other Feature Feedback, Questions, & Ideas]
<p><a href="https://phpseclib.github.io/docs/why" rel="noopener nofollow ugc">https://phpseclib.github.io/docs/why</a> is 301 redirecting to <a href="https://docs.phpseclib.org/why" rel="noopener nofollow ugc">https://docs.phpseclib.org/why</a> and idk why.</p>
<p>I used to have a phpseclib/docs repo that was setup to publish to github pages but I disabled that since that was wanting to publish to <code>docs.phpseclib.org</code>. But now that that’s disabled and the 301 redirect is still happening idk what’s up.</p>
<p>Any ideas?</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/terrafrost">@terrafrost</a>, welcome to the GitHub Support Community!</p>
<p>I’m not seeing this on my end so it’s likely cached in your browser. 301 redirects stick around in your browser’s indefinitely so emptying your cache should stop this happening for you.</p>
